multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -133,17 +135,18 @@ coreos:
       content: |
         [Unit]
         Description=decrypt kubelet tls assets using amazon kms
-        Before=kubelet.service
-        After=docker.service
-        Requires=docker.service
+        Before=kubelet.service flanneld.service
+        After=early-docker.service
+        Requires=early-docker.service
```

Comment:
I'm assuming this is because this is being executed via docker run, but I'd prefer we used rkt